README.md

```diff
@@ -24,178 +24,133 @@ npm install @ice/store --save
 
 `icestore` is a lightweight React state management library based on hooks. It has the following core features:
 
-* **Minimal API**: Contains 5 APIs, which is easily learnable in 5 minutes.
-* **Typescript Support**: Provide complete type definitions to support IntelliSense in VSCode.
-* **Predictable**: Uses unidirectional data flow (similar to Redux) and allows state mutation only inside actions, allowing data flow to be traced easily.
-* **Optimal performance**: Decreases the number of view components that rerender when the state changes by creating multiple stores.
-* **Built in async status**: Records loading and error status of async actions, simplifying the rendering logic in the view layer.
+* **Minimal & Familiar API**: No additional learning costs, just use React Hooks as intended.
+* **Enough & Extensible**: Meet 80% of scenarios, another 20%? Middleware mechanism can be used to meet the requirements.
```

Comment:
Cover 80% most common usage scenes and extend the remaining  20% scenes with builtin middleware mechanism.

Reply:
done